helm/crds/notification.kubesphere.io_dingtalkconfigs.yaml

```diff
@@ -38,97 +38,39 @@ spec:
           spec:
             description: DingTalkConfigSpec defines the desired state of DingTalkConfig
             properties:
-              chatbot:
-                description: Be careful, a ChatBot only can send 20 message per minute.
+              appkey:
+                description: The key of the application which sending message.
```

Comment:
The key of the application which sending message. =>
The key of the application with which to send messages.